src/Content/VersionId.php

```diff
@@ -68,18 +66,14 @@ public static function changes(): static
 	}
 
 	/**
-	 * Returns the default version id for the model
+	 * List of available version ids
 	 */
-	public static function default(ModelWithContent $model): static
+	public static function list(): array
 	{
-		if (
-			$model instanceof Page === true &&
-			$model->isDraft() === true
-		) {
-			return VersionId::changes();
-		}
-
-		return VersionId::published();
+		return [
+			static::published(),
```

Comment:
You are right. Maybe the best would be `VersionId::latest()` This would also still be pretty useful as soon as we have multiple versions. But I would love to not handle that in this PR to be honest. It would get quite massive.

Reply:
I'm ok with solving this later. Can we add it to the Todo list?

Reply:
I've just added a todo in Notion.